docs/entities/OfferReview.md

```diff
@@ -0,0 +1,24 @@
+# OfferReview
+
+## Descripción
+Representación de una resenya en la plataforma.
+
+## Propiedades
+* id: Identificador único de la reseña.
+* buyerId: Identificador único del comprador.
+* userImageUrl: URL de la imagen del usuario que escribió la reseña.
+* content: Texto de la reseña, que describe la experiencia del comprador con la oferta.
```

Comment:
If we know this is going to be text only I would close the possibilities by naming it as "message", "body" or even "text".